src/devices/Interop/Unix/ThreadHelper.cs

```diff
@@ -0,0 +1,65 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+namespace System.Interop
+{
+    internal class ThreadHelper
```

Comment:
NIT: should the class itself be static?

Reply:
Yes, but it is not big deal as it is internal.